mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -1811,6 +1869,7 @@ async fn recv_swap_v2_msg<T>(
     let wait_until = started + timeout;
     loop {
         Timer::sleep(1.).await;
+        // TODO: db_id
         let swap_ctx = SwapsContext::from_ctx(&ctx).unwrap();
```

Comment:
I don't think we need `db_id` here. Swap msg storage is an in-memory storage where the key is the swap uuid.